src/rational/q/properties.rs

```diff
@@ -0,0 +1,126 @@
+// Copyright © 2023 Niklas Siemer
+//
+// This file is part of qFALL-math.
+//
+// qFALL-math is free software: you can redistribute it and/or modify it under
+// the terms of the Mozilla Public License Version 2.0 as published by the
+// Mozilla Foundation. See <https://mozilla.org/en-US/MPL/2.0/>.
+
+//! This module includes functionality about properties of [`Q`] instances.
+
+use super::Q;
+use flint_sys::fmpq::{fmpq_abs, fmpq_inv};
+
+impl Q {
+    /// Returns the given [`Q`] instance with its absolute value.
+    ///
+    /// # Example
+    /// ```
+    /// use qfall_math::rational::Q;
+    /// let mut value = Q::from(-1);
+    ///
+    /// let value = value.abs();
+    ///
+    /// assert_eq!(Q::ONE, value);
+    /// ```
+    pub fn abs(mut self) -> Self {
+        unsafe {
+            fmpq_abs(&mut self.value, &self.value);
+        }
+        self
+    }
+
+    /// Returns the inverse of `self` as a fresh [`Q`] instance.
+    ///
+    /// As the inverse of `0` is undefined, it returns `None` in case `self == 0`.
+    ///
+    /// # Example
+    /// ```
+    /// use qfall_math::rational::Q;
+    /// let value = Q::from(4);
+    ///
+    /// let inverse = value.inv().unwrap();
+    ///
+    /// assert_eq!(Q::try_from((&1, &4)).unwrap(), inverse);
+    /// ```
+    pub fn inv(&self) -> Option<Q> {
+        if self == &Q::ZERO {
+            return None;
+        }
```

Comment:
For this case we theoretically could use `fmpq_is_zero`. We might have to look which is more efficient, but I would  guess using the dedicated might be more efficient.